app/presenters/relationship_label.rb

```diff
@@ -129,6 +131,6 @@ def default_description
   end
 
   def amount_display
-    ActiveSupport::NumberHelper.number_to_currency(amount, precision: 0)
+    number_to_currency(amount, unit: currency.upcase, precision: 0, format: '%n %u')
```

Comment:
I think I have a preference for using the symbols -- $/€instead of USD/EUR -- but we can do a quick poll of some users to see which format others prefer.